lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,19 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.last.nil?
+            ""
+          else
+            "#{comments.last.body.inspect}"
+          end
+        end
+
+        def approved?
```

Comment:
The current way it is structured is correct logically provided that we don't want to check the last comment if we are in bit bucket.